examples/class-definition/class_definition.bal

```diff
@@ -0,0 +1,30 @@
+import ballerina/io;
+
+// Defines a class called `Person`. It contains `public`, `private`, and module-level visible fields along with their types.
+// For fields, the default value must be specified if an explicit object initializer method is not defined.
+class Person {
+    public string name = "";
+    public int age = 0;
+    // If you need to have a field with its own type, you need to make that field defaultable
+    // (i.e., able to specify a default value).
+    // Here, the `parent` field is made defaultable by allowing it to have nil as a value.
```

Comment:
Defaultable fields is sort of a casual, internal term we use because that's what we called it before the spec defined it right? IMHO, it's better to say that if one wants to have a field of the type being defined, one needs to make the the type of the field an optional type.

Reply:
This file is just the old object BBE with syntax and description altered. Since the file was renamed to `class_definition.bal` and new file was added with name `object.bal` in the same PR, the diff seems to think this is a new file.

Agreed that the term making a field defautable sounds odd, will send a PR fixing this.